cmd/cluster/dynatrace.go

```diff
@@ -0,0 +1,130 @@
+package cluster
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strings"
+
+	bplogin "github.com/openshift/backplane-cli/cmd/ocm-backplane/login"
+	bpconfig "github.com/openshift/backplane-cli/pkg/cli/config"
+	ocmutils "github.com/openshift/osdctl/pkg/utils"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+)
+
+var DynatraceURLStaging = map[string]string{
+	"us": "https://hrm15629.apps.dynatrace.com/",
+	"eu": "https://wmb10400.apps.dynatrace.com/",
+}
+
+var DynatraceURLProduction = map[string]string{
+	"us": "https://jgn20300.apps.dynatrace.com/",
+	"eu": "https://vap62935.apps.dynatrace.com/",
+	"ap": "https://zwz85475.apps.dynatrace.com/",
+}
+
+func newCmdDynatraceURL() *cobra.Command {
+	orgIdCmd := &cobra.Command{
+		Use:   "dynatrace CLUSTER_ID",
+		Short: "Get the Dyntrace Tenant URL for a given MC cluster",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdutil.CheckErr(fetchDetails(args[0]))
+		},
+	}
+	return orgIdCmd
+}
+
+func fetchDetails(clusterID string) error {
+	if err := ocmutils.IsValidClusterKey(clusterID); err != nil {
+		return err
+	}
+	connection, err := ocmutils.CreateConnection()
+	if err != nil {
+		return err
+	}
+	defer connection.Close()
+
+	cluster, err := ocmutils.GetCluster(connection, clusterID)
+	if err != nil {
+		return err
+	}
+	region := cluster.Region().ID()
+	if !strings.HasPrefix(cluster.Name(), "hs-mc-") {
+		return fmt.Errorf("cluster is Not a Management Cluster")
+	}
+	url, err := GetDynatraceURLFromCluster(cluster.ID())
+	if err != nil {
+		OCMEnv := ocmutils.GetCurrentOCMEnv(connection)
+		url = getDynatraceURLFromRegion(region, OCMEnv)
+		if url == "" {
+			return fmt.Errorf("the Dynatrace Environemnt URL could not be determined")
+		}
```

Comment:
Related to another comment, but could we just refer users to an SOP, e.g. https://github.com/openshift/ops-sop/blob/master/dynatrace/README.md ?